pimd/pim_cmd.c

```diff
@@ -11147,14 +9800,12 @@ void pim_cmd_init(void)
 	install_element(ENABLE_NODE, &no_debug_pim_mlag_cmd);
 	install_element(ENABLE_NODE, &debug_pim_vxlan_cmd);
 	install_element(ENABLE_NODE, &no_debug_pim_vxlan_cmd);
-#if PIM_IPV != 6
```

Comment:
why is this removed?

Reply:
The whole commit was cherry-picked from your changes, i didn't change anything